static/js/place/place.ts

```diff
@@ -279,7 +278,6 @@ function renderPage(): void {
           categoryStrings: data.categories,
           locale,
           highlight: data.highlight,
-          summaryText,
```

Comment:
Can we remove `summaryText` as a prop from `MainPanePropType` altogether now?

Reply:
Done!